js/plugins/tooltips.js

```diff
@@ -93,6 +93,12 @@ export function install(Vue) {
                 }
             }, popover));
 
+            // Transclude child content if found
+            const content = this.el.querySelector('[popover-content]');
```

Comment:
As you wish, consistency is more important in that case and I don't know what is the current best practice for others vuejs files.